.github/workflows/pull_request_build.yml

```diff
@@ -51,32 +51,32 @@ jobs:
       env:
         SONAR_TOKEN: "e5dbb8f62a0b8bf02dbbcf4c9fd745c31b63f4b3"
 
-#    - name: Run Example Project Test with JDK8
-#      run: |
-#        cd example-projects/example-project
-#        mvn clean package --> jsonld issue
+    - name: Run Example Project Test with JDK8
```

Comment:
Again the commit message lacks the context, so it's unclear whether tests as added to the code base, or just to the CI scripts (i.e. their are not "added" but "run").

Reply:
The commit message mysteriously talks about "an issue with JsonId", but it should explain in this context what exactly the problem with JsonId was.